core/src/main/java/io/jenkins/servlet/ServletContextWrapper.java

```diff
@@ -0,0 +1,692 @@
+            }
+
+            @Override
+            public void setSessionTimeout(int sessionTimeout) {
+                from.setSessionTimeout(sessionTimeout);
+            }
+
+            @Override
+            public String getRequestCharacterEncoding() {
+                return from.getRequestCharacterEncoding();
+            }
+
+            @Override
+            public void setRequestCharacterEncoding(String encoding) {
+                from.setRequestCharacterEncoding(encoding);
+            }
+
+            @Override
+            public String getResponseCharacterEncoding() {
+                return from.getResponseCharacterEncoding();
+            }
+
+            @Override
+            public void setResponseCharacterEncoding(String encoding) {
+                from.setResponseCharacterEncoding(encoding);
+            }
+        };
+    }
+
+    public static ServletContext fromJakartServletContext(jakarta.servlet.ServletContext from) {
+        Objects.requireNonNull(from);
+        return new ServletContext() {
+            @Override
+            public String getContextPath() {
+                return from.getContextPath();
+            }
+
+            @Override
+            public ServletContext getContext(String uripath) {
+                return fromJakartServletContext(from.getContext(uripath));
+            }
+
+            @Override
+            public int getMajorVersion() {
+                return from.getMajorVersion();
+            }
+
+            @Override
+            public int getMinorVersion() {
+                return from.getMinorVersion();
+            }
+
+            @Override
+            public int getEffectiveMajorVersion() {
+                return from.getEffectiveMajorVersion();
+            }
+
+            @Override
+            public int getEffectiveMinorVersion() {
+                return from.getEffectiveMinorVersion();
+            }
+
+            @Override
+            public String getMimeType(String file) {
+                return from.getMimeType(file);
+            }
+
+            @Override
+            public Set<String> getResourcePaths(String path) {
+                return from.getResourcePaths(path);
+            }
+
+            @Override
+            public URL getResource(String path) throws MalformedURLException {
+                return from.getResource(path);
+            }
+
+            @Override
+            public InputStream getResourceAsStream(String path) {
+                return from.getResourceAsStream(path);
+            }
+
+            @Override
+            public RequestDispatcher getRequestDispatcher(String path) {
+                return RequestDispatcherWrapper.fromJakartaRequestDispatcher(from.getRequestDispatcher(path));
+            }
+
+            @Override
+            public RequestDispatcher getNamedDispatcher(String name) {
+                return RequestDispatcherWrapper.fromJakartaRequestDispatcher(from.getNamedDispatcher(name));
```

Comment:
The method is documented to potentially return `null` so this should account for that.

Reply:
Fixed in the latest revision.